plugins/googlecloudplatform/bq-schema/v1.1.0/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1.6
+FROM --platform=$BUILDPLATFORM golang:1.22.1-bookworm AS build
+
+ARG TARGETOS TARGETARCH
+ENV CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH
+
+RUN --mount=type=cache,target=/go/pkg/mod \
+    go install -ldflags="-s -w" -trimpath github.com/GoogleCloudPlatform/protoc-gen-bq-schema@v1.1.0 \
+    && mv /go/bin/${GOOS}_${GOARCH}/protoc-gen-bq-schema /go/bin/protoc-gen-bq-schema || true
+
+FROM scratch
+COPY --from=build --link --chown=root:root /etc/passwd /etc/passwd
+COPY --from=build --link /go/bin/protoc-gen-bq-schema /
+USER nobody
+ENTRYPOINT [ "/protoc-gen-bq-schema", "-logtostderr=true" ]
```

Comment:
A bit surprised that logging to stderr is not the default (since otherwise it would emit a malformed generator response).

Reply:
I'm not too familiar with [golang/glog](https://github.com/golang/glog), but if I removed this it would fail for me locally. Let me try to remove it and see if it fails in CI.